Okta.Sdk.Abstractions/UserAgentBuilder.cs

```diff
@@ -44,7 +44,7 @@ private string Generate()
         {
             var sdkToken = $"{_oktaSdkUserAgentName}/{GetSdkVersion()}";
 
-            var runtimeToken = $"runtime/{Sanitize(RuntimeInformation.FrameworkDescription)}";
+            var runtimeToken = $"runtime/{UserAgentHelper.GetRuntimeVersion()}";
```

Comment:
Are there usages of the overloads to UserAgentHelper.GetRuntimeVersion?  There are variant invocations available (UserAgent.GetRuntimeVersion("some description"), etc.), where would those variants be used?

Reply:
The whole point of allowing to pass a param here is to mock possible values for `RuntimeInformation.FrameworkDescription` and test string manipulation. In reality, we're not gonna pass any custom descriptions.